engine/src/main/java/org/terasology/rendering/dag/FBOStateChange.java

```diff
@@ -15,19 +15,18 @@
  */
 package org.terasology.rendering.dag;
 
-import java.util.HashMap;
-import static org.lwjgl.opengl.GL11.GL_FILL;
-
 /**
  * TODO: Add javadocs
  */
-class DefaultStateChangeStorage extends HashMap<Class<? extends StateChange>, StateChange> {
+public class FBOStateChange extends AbstractStateChange<Integer> {
```

Comment:
I'd rename this to BindFBOStateChange. Or perhaps, I dare say, even just BindFBO.